Assets/Script/Projectile.cs

```diff
@@ -1,27 +1,44 @@
-using System.Collections;
 using UnityEngine;
 
 [RequireComponent(typeof(Rigidbody))]
 public class Projectile : MonoBehaviour
 {
     private Rigidbody rb;
-    private float velocity;
+    [SerializeField] private float velocity;
+    [SerializeField] private float damage;
+    [SerializeField] private GameObject explosion;
+
     private void Start()
     {
         Setup();
         StartMoving();
+        Destroy(gameObject, 5f);
```

Comment:
Consider using constants for numbers like this, e.g.

`private const float secondsToLive = 5f;`